db/schema.rb

```diff
@@ -138,7 +138,7 @@
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
     t.integer  "belongs_to"
-    t.boolean  "is_common",  default: false
+    t.boolean  "is_common",  default: false, index: {name: "index_skills_on_is_common"}
```

Comment:
super super minor nit - single vs double quotes to prevent interpolation and how do you feel about spacing in here? I generally prefer `{ name: 'index_skills_on_is_common' }`.